synapse/lib/stormtypes.py

```diff
@@ -7645,6 +7679,14 @@ class View(Prim):
                   ),
                   'returns': {'type': 'int', 'desc': 'The count of nodes.', }}},
 
+        {'name': 'iterPropUniqValues',
```

Comment:
quick goback: we often use `getFooBars()` vs `iterFooBars()` even for generators. Do we want to decide on a convention and begin converting other APIs for syn 3.x?

Reply:
updated to `iterPropValues`